### PR TITLE
Partial Filipino Translation - 1 111 translated words

### PR DESCRIPTION
**Pull request by mind-blast**
Translating revision-selection.asc file with 1,111 translated words
Note : Code text and computer generate code are not included as translated words

### CODE REVIEW

File: book/07-git-tools/sections/revision-selection.asc [hunk 1]
```diff
@@ -1,19 +1,19 @@
 [[_revision_selection]]
-=== Revision Selection
+=== Pagbabago ng Pinili
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: you could use ``Pagpipili ng Rebisyon``

File: book/07-git-tools/sections/revision-selection.asc [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
-Git allows you to refer to a set of commits or a range of commits in several ways.
-They aren’t necessarily obvious but are helpful to know.
+Ang Git ay nagpapahintulot sa iyo upang sumangguni sa isang itinakdang mga commit o isang lawak ng mga commit sa maraming mga paraan.
+Hindi nila kinakailangan na halata ngunit nakakatulong upang malaman.
 
-==== Single Revisions
+==== Solong mga Pagbabago
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: you could use `Solong mga Rebisyon`

File: book/07-git-tools/sections/revision-selection.asc [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
 
-You can obviously refer to any single commit by its full, 40-character SHA-1 hash, but there are more human-friendly ways to refer to commits as well.
-This section outlines the various ways you can refer to any commit.
+Maaari mong malinaw na i-refer ang anumang solong commit sa buong ito, 40-karakter na SHA-1 hash, ngunit may mga higit pa naa tao-palakaibigan na mga paraan upang i-refer sa mga commit din.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: - `na pantaong-palakaibigan`
- upang sumangguni rin sa mga commit

File: book/07-git-tools/sections/revision-selection.asc [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
+Ang seksyon na ito ng mga outline ng  iba't ibang mga paraan na maaari mong i-refer sa anumang commit.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: - remove extra space between `ng  iba't`
- change i-refer to isangguni

File: book/07-git-tools/sections/revision-selection.asc [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
 
-==== Short SHA-1
+==== Ang Maikling SHA-1
 
-Git is smart enough to figure out what commit you're referring to if you provide the first few characters of the SHA-1 hash, as long as that partial hash is at least four characters long and unambiguous; that is, no other object in the object database can have a hash that begins with the same prefix.
+Ang Git ay isang matalino ay sapat na matalino upang malaman kung anong commit ang iyong nare-refer na kung bigyan mo ang mga unang ilang mga karakter ng SHA-1 hash, hangga't ang bahagyang hash ay hindi bababa sa apat na mga karakter na mahaba at hindi malabo; yan ay, walang ibang bagay sa database ng bagay na maaaring magkakaroon ng hash na nagmumula ng parehong prefix.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: - Ang Git ay sapat na matalino
- hindi bababa sa apat na mga karakter ang haba

File: book/07-git-tools/sections/revision-selection.asc [hunk 6]
```diff
@@ -59,49 +59,49 @@ ca82a6d changed the version number
 a11bef0 first commit
 ----
 
-Generally, eight to ten characters are more than enough to be unique within a project.
-For example, as of October 2017, the Linux kernel (which is a fairly sizable project) has over 700,000 commits and almost six million objects, with no two objects whose SHA-1s are identical in the first 11 characters.
+Sa pangkalahatan, walo sa sampung mga karakter ay higit pa sa sapat upang maging natatanging sa loob ng isang proyekto.
+Halimbawa, sa Oktubre 2017, ang Linux na kernel (na kung saan ay isang medyo malaki na proyekto) ay may higit sa 700,000 na mga commit at halos anim na milyong na mga bagay , na may walang dalang mga bagay na magkatulad ang mga SHA-1 ay magkapareho sa unang 11 na mga karakter.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: remove extra space in `bagay ,`

File: book/07-git-tools/sections/revision-selection.asc [hunk 7]
```diff
@@ -59,49 +59,49 @@ ca82a6d changed the version number
...
 
-[NOTE]
-.A SHORT NOTE ABOUT SHA-1
+[TANDAAN]
+.ISANG MAIKLING PAALALA TUNGKOL SA SHA-1
 ====
 
-A lot of people become concerned at some point that they will, by random happenstance, have two distinct objects in their repository that hash to the same SHA-1 value.
-What then?
+Maraming tayo ay magiging nababahala sa isang punto na sila ay, sa pamamagitan ng random na happenstance, mayroong dalawang magkakaibang bagay sa kanilang repositoryo na ang hash ay pareho sa SHA-1 na halaga.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: - Maraming tao
- `random` to `sapalaran`
- `happenstance` to `pagkakataon`

File: book/07-git-tools/sections/revision-selection.asc [hunk 8]
```diff
@@ -59,49 +59,49 @@ ca82a6d changed the version number
...
+Ano ngayon?
 
-If you do happen to commit an object that hashes to the same SHA-1 value as a previous _different_ object in your repository, Git will see the previous object already in your Git database, assume it was already written and simply reuse it.
-If you try to check out that object again at some point, you’ll always get the data of the first object.
+Kung mangyari na ikaw ay mag-commit sa isang bagay na nag-hash sa parehong SHA-1 na halaga bilang isang nakaraang _ibang_ bagay sa iyong repositoryo, Ang Git ay makakita ng nakaraan na bagay na nasa iyong Git na database, ipagpalagay na ito ay nakasulat na at mula lang gamitin ito.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: change `mula` to `muli`

File: book/07-git-tools/sections/revision-selection.asc [hunk 9]
```diff
@@ -59,49 +59,49 @@ ca82a6d changed the version number
...
+Kung subukan mong tingnan ang bagay na iyon muli sa isang punto, ikaw ay makakuha lagi ng datos sa unang bagay.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: makakakuha

File: book/07-git-tools/sections/revision-selection.asc [hunk 10]
```diff
@@ -59,49 +59,49 @@ ca82a6d changed the version number
...
 
-However, you should be aware of how ridiculously unlikely this scenario is.
-The SHA-1 digest is 20 bytes or 160 bits.
-The number of randomly hashed objects needed to ensure a 50% probability of a single collision is about 2^80^
-(the formula for determining collision probability is `p = (n(n-1)/2) * (1/2^160))`. 2^80^
-is 1.2 x 10^24^
-or 1 million billion billion.
-That’s 1,200 times the number of grains of sand on the earth.
+Gayunpaman, dapat mong malaman kung gaano katawa ang pagkawala sa sitwasyon na ito.
+Ang SHA-1 na digest ay 20 bytes oo 160 bits.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: `oo` to `o`

File: book/07-git-tools/sections/revision-selection.asc [hunk 11]
```diff
@@ -59,49 +59,49 @@ ca82a6d changed the version number
...
 
-Here’s an example to give you an idea of what it would take to get a SHA-1 collision.
-If all 6.5 billion humans on Earth were programming, and every second, each one was producing code that was the equivalent of the entire Linux kernel history (3.6 million Git objects) and pushing it into one enormous Git repository, it would take roughly 2 years until that repository contained enough objects to have a 50% probability of a single SHA-1 object collision.
-Thus, a SHA-1 collision is less likely than every member of your programming team being attacked and killed by wolves in unrelated incidents on the same night.
+Narito ang isang halibawa na magbibigay sa iyo ng ideya kung ano ang kukuha nitong SHA-1 na banggaan
+Kung lahat sa 6.5 na bilyon na mga tayo sa Lupa ay nag-programming, sa bawat segundo, sa bawat isa ay gumagawa ng code na katumbas sa buong Linux na kernel na kasaysayan(3.6 na milyon Git na mga bagay) at itinutulak into sa isang napakalaking Git na repositoryo, ito ay kukuha ng halos dalawang taon hanggang ang repositoryong ito ay maglalaman ng sapat na mga bagay upang magkaroon ng 50% na posibilidad sa isang SHA-1 na bagay na banggaan.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: - change `nag-programming` to `nag-program`
- 3.6 na milyong

File: book/07-git-tools/sections/revision-selection.asc [hunk 12]
```diff
@@ -59,49 +59,49 @@ ca82a6d changed the version number
...
+Kaya naman, ang SHA-1 na banggaan ay mas malamang kaysa sa bawat miyembro sa iyong programming na koponan na inatake o napatay ng mga lobo sa hindi nauugnay na insidente sa parehong gabi
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: parehong gabi.

File: book/07-git-tools/sections/revision-selection.asc [hunk 13]
```diff
@@ -59,49 +59,49 @@ ca82a6d changed the version number
...
 
 ====
 
 [[_branch_references]]
-==== Branch References
+==== Mga Reperensiya ng Branch
 
-One straightforward way to refer to a particular commit is if it's the commit at the tip of a branch; in that case, you can simply use the branch name in any Git command that expects a reference to a commit.
-For instance, if you want to examine the last commit object on a branch, the following commands are equivalent, assuming that the `topic1` branch points to commit `ca82a6d...`:
+Isang tapat na paraan upang i-refer ang isang partikular na commit na kung itong commit na nasa tuktuk na isang branch; sa ganitong kaso, maaari mo lamang gamitin ang pangalan ng branch sa anumang Git na utos na inaasahan ng isang reperensiya sa isang commit.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: tuktok

File: book/07-git-tools/sections/revision-selection.asc [hunk 14]
```diff
@@ -110,11 +110,11 @@ ca82a6dff817ec66f44342007202690a93763949
 ----
 
 [[_git_reflog]]
-==== RefLog Shortnames
+==== Mga Shortname ng RefLog
 
-One of the things Git does in the background while you’re working away is keep a ``reflog'' -- a log of where your HEAD and branch references have been for the last few months.
+Isa sa mga bagay ng Git ay gumawa ng background habang ikaw ay nagtratrabaho ng malayo ay magtago ng ``reflog'' -- isang log ona kung saan ang iyong HEAD at branch na mga reperensiya ay para sa huling ilang mga buwan.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: isang log na

File: book/07-git-tools/sections/revision-selection.asc [hunk 15]
```diff
@@ -128,27 +128,27 @@ d921970 HEAD@{1}: merge phedders/rdocs: Merge made by the 'recursive' stategy.
 7e05da5 HEAD@{6}: rebase -i (pick): updating HEAD
 ----
 
-Every time your branch tip is updated for any reason, Git stores that information for you in this temporary history.
-You can use your reflog data to refer to older commits as well.
-For example, if you want to see the fifth prior value of the HEAD of your repository, you can use the `@{5}` reference that you see in the reflog output:
+Sa tuwing iyong branch tip ay na-update sa anumang rason, ang Git ay nag-iimbak ng impormasyon para sa iyong temporaryong kasaysayan.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc:131]: pansamantalang kasaysayan

File: book/07-git-tools/sections/revision-selection.asc [hunk 16]
```diff
@@ -128,27 +128,27 @@ d921970 HEAD@{1}: merge phedders/rdocs: Merge made by the 'recursive' stategy.
...
+Maaari mong gamitin ang iyong reflog na datos upang i-refer ang matandang mga commit din.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: i-refer din ang mga lumang mga commit.

File: book/07-git-tools/sections/revision-selection.asc [hunk 17]
```diff
@@ -128,27 +128,27 @@ d921970 HEAD@{1}: merge phedders/rdocs: Merge made by the 'recursive' stategy.
...
+Halimbawa, kung ikaw ay gustong makakita ng pangliwang bago ang halaga sa HEAD ng iyong repositoryo, maaari mong gamitin ang `@{5}` na reperensiya na nakikita mo sa reflog output:
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: panglimang

File: book/07-git-tools/sections/revision-selection.asc [hunk 18]
```diff
@@ -128,27 +128,27 @@ d921970 HEAD@{1}: merge phedders/rdocs: Merge made by the 'recursive' stategy.
...
 
 [source,console]
 ----
 $ git show master@{yesterday}
 ----
 
-That would show you where tip of your `master` branch was yesterday.
-This technique only works for data that’s still in your reflog, so you can’t use it to look for commits older than a few months.
+Iyon ay magpapakita sa iyo kung saan ang tip ng iyong `master` na branch kahapon.
+Ang pamamaraan na ito ay gumagana lamang para sa datos na nasa iyong reflog pa, kaya ikaw ay hindi makagamit nito ng mga commit na mas matanda na sa ilang mga buwan.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: mas luma pa sa ilang mga buwan

File: book/07-git-tools/sections/revision-selection.asc [hunk 19]
```diff
@@ -170,14 +170,14 @@ Date:   Thu Dec 11 15:08:43 2008 -0800
     Merge commit 'phedders/rdocs'
 ----
 
-It’s important to note that reflog information is strictly local -- it’s a log only of what _you've_ done in _your_ repository.
-The references won’t be the same on someone else’s copy of the repository; also, right after you initially clone a repository, you'll have an empty reflog, as no activity has occurred yet in your repository.
-Running `git show HEAD@{2.months.ago}` will show you the matching commit only if you cloned the project at least two months ago -- if you cloned it any more recently than that, you'll see only your first local commit.
+Mahalagang tandaan na ang reflog na impormasyon ay mahigpit na lokal -- ito ay isang log lamang kung ano _ang iyong_ nagawa sa _iyong_ repositoryo.
+Ang mga reperensiya ay hindi pareho sa ibang kopya ng repositoryo; din, pagkatapos ka mag-clone ng repositoryo, ikaw ay magkakarooon ng walang laman na reflog, na bilang walang aktibidad na naganap pa sa iyong repositoryo.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: change `din` to `at saka`

File: book/07-git-tools/sections/revision-selection.asc [hunk 20]
```diff
@@ -170,14 +170,14 @@ Date:   Thu Dec 11 15:08:43 2008 -0800
...
+Pagtatakbo ng `git show HEAD@{2.months.ago}` ay ipapakita sa iyo ang pagtutugma ng commit lamang kung ikaw ay nag-clone ng proyekto na hindi bababa sa dalawang buwan na nakalipas -- kung na-clone mo itong mas kamakailan pa kaysa sa iyon, makikita mo lamang ang iyong unang lokal na commit.
 
 [TIP]
-.Think of the reflog as Git's version of shell history
+.Isipin ang reflog bilang bersyon ng Git sa kasaysayan ng shell
 ====
-If you have a UNIX or Linux background, you can think of the reflog as Git's version of shell history, which emphasizes that what's there is clearly relevant only for you and your ``session'', and has nothing to do with anyone else who might be working on the same machine.
+Kung ikaw ay mayroong UNIX o Linux na background, maaari mong isipin na ang iyong reflog bilang bersyon ng Git sa kasaysayan ng shell, na nagpapahiwatig na kung ano ang naroroon ay malinaw na may kaugnayan lamang para sa iyo at iyong ``sesyon'', at walang kinalaman sa sinuman na maaaring nagtratrabaho sa parehong makina.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: change `background` to `karanasan`

**Review by josteinfrancois — COMMENTED** [review]
Hello @robin-ho. I have finished reviewing your work. Good job. I only have a few suggestions. Please refer to the comments I provided. Thanks.

File: book/07-git-tools/sections/revision-selection.asc (hunk 1 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 2 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 3 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 4 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 5 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 6 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 7 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 8 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 9 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 10 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 11 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 12 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 13 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 14 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 15 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc:131]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 16 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 17 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 18 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 19 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 20 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

**Comment by mind-blast** [comment]
done change all suggested words. Thank you for proof read.

**Pull request merged**